pandas/io/gcs.py

```diff
@@ -0,0 +1,26 @@
+""" GCS support for remote file interactivity """
+from pandas import compat
+try:
+    from google.auth.exceptions import GoogleAuthError
+    import gcsfs
+except ImportError:
+    raise ImportError("The gcsfs library is required to handle GCS files")
+
+
+def get_filepath_or_buffer(filepath_or_buffer, encoding=None,
+                           compression=None, mode=None):
+
+    if mode is None:
+        mode = 'rb'
+
+    gcsfs_logger_disabled = gcsfs.core.logger.disabled
+    gcsfs.core.logger.disabled = True
```

Comment:
Why disable this? I'd rather leave that up to the user if possible.

Reply:
This was probably an overly complicated solution; the problem I was facing was that `gcsfs` retries several times if it fails to authenticate, and each failure prints a long traceback. But if that's the behavior of the library then I guess there's no sense in using hacks to get around it. For now I removed all of the logging magic.